### ISSUE TITLE
[Release-1.21] - k3s secrets-encrypt status is not able to connect to API not bound to 127.0.0.1

### ISSUE THREAD

**Issue by dereknola** [k3s-io/k3s#5052]
Backport fix for k3s secrets-encrypt status is not able to connect to API not bound to 127.0.0.1

* #5006

**Comment by ShylajaDevadiga** [comment]
Reproduced the issue on k3s v1.21.9+k3s1 passing non-default bind-address 0.0.0.0
```
sudo k3s server --cluster-init --secrets-encryption --bind-address=<ip>
```
Error while fetching the status 
```
$ sudo k3s secrets-encrypt status
FATA[2022-02-14T20:37:02.887599501Z] failed to get CA certs: Get "https://127.0.0.1:6443/cacerts": dial tcp 127.0.0.1:6443: connect: connection refused 
```

- Validated fix on **release-1.21** branch using commit id **c88374a3adf235c5026a865c4ad02a33cfe38188** passing the server flag

```
$ sudo k3s secrets-encrypt status --server https://<ip>:6443
Encryption Status: Enabled
Current Rotation Stage: start
Server Encryption Hashes: All hashes match

Active  Key Type  Name
------  --------  ----
 *      AES-CBC   aescbckey
```
Validated using config.yaml
```
$ cat /etc/rancher/k3s/config.yaml 
bind-address: <ip>
secrets-encryption: true
```

**Comment by ShylajaDevadiga** [comment]
Validated on k3s version v1.21.10-rc1+k3s1 (471f5eb3)
```
sudo /usr/local/bin/k3s secrets-encrypt status --server https://<ip>:6443
Encryption Status: Enabled
Current Rotation Stage: start
Server Encryption Hashes: All hashes match

Active  Key Type  Name
------  --------  ----
 *      AES-CBC   aescbckey
 ```
